gpt4all-api/README.md

```diff
@@ -8,9 +8,14 @@ The following tutorial assumes that you have checked out this repo and cd'd into
 
 ### Starting the app
 
-First change your working directory to `gpt4all/gpt4all-api`.
+First, decide what LLM model you'll use in your API and be sure to download or setup your `~/models` folder with the LLM models you'd like to use with the API, like this:
+```bash
+export MODEL_BIN=llama-2-7b-chat.ggmlv3.q8_0.bin
+mkdir -p ~/models
+wget https://huggingface.co/TheBloke/Llama-2-7B-Chat-GGML/resolve/main/${MODEL_BIN} -P ~/models
```

Comment:
Should we add these step in deploy script here https://github.com/notasecret/auto-coach-server/blob/strava/quickstart_deploy.sh?